fastlane/lanes/release_management_in_ci.rb

```diff
@@ -0,0 +1,65 @@
+BUILDKITE_ORGANIZATION = 'automattic'.freeze
+BUILDKITE_PIPELINE = 'wordpress-android'.freeze
+platform :android do
+  #####################################################################################
+  # Triggers for Buildkite
+  #####################################################################################
+  lane :trigger_code_freeze_in_ci do |options|
+    buildkite_trigger_build(
+      buildkite_organization: BUILDKITE_ORGANIZATION,
+      buildkite_pipeline: BUILDKITE_PIPELINE,
+      branch: 'trunk',
+      pipeline_file: 'code-freeze.yml',
+      message: 'Code Freeze in CI'
+    )
+  end
+
+  lane :trigger_complete_code_freeze_in_ci do |options|
+    release_version = options[:release_version]
+    buildkite_trigger_build(
+      buildkite_organization: BUILDKITE_ORGANIZATION,
+      buildkite_pipeline: BUILDKITE_PIPELINE,
+      branch: "release/#{release_version}",
+      pipeline_file: 'complete-code-freeze.yml',
+      message: 'Complete Code Freeze in CI',
+      environment: { RELEASE_VERSION: release_version }
+    )
+  end
+
+  lane :trigger_finalize_release_in_ci do |options|
+    release_version = options[:release_version]
+    buildkite_trigger_build(
+      buildkite_organization: BUILDKITE_ORGANIZATION,
+      buildkite_pipeline: BUILDKITE_PIPELINE,
+      branch: "release/#{release_version}",
+      pipeline_file: 'finalize-release.yml',
+      message: 'Finalize release',
```

Comment:
The previous messages use "Title Case":

```suggestion
      message: 'Finalize Release',
```